src/renderer/drawers/Provider.js

```diff
@@ -0,0 +1,51 @@
+/* @flow */
+import React, { useReducer, useEffect, useCallback } from "react";
+
+type State = {
+  Component: ?React$ComponentType<*>,
+  props?: *,
+  open: boolean,
+};
+
+// actions
+// it makes them available and current from connector events handlers
+export let setDrawer: (Component?: *, props?: *) => void = () => {};
```

Comment:
Why not providing a hook here ?

like:
```
export const useDrawer = () => {
  const context = useContext(DrawerContext);
  if (context === undefined) {
    throw new Error(`useDrawer must be used within a DrawerProvider`);
  }
  return context;
};
```

The solution here is a bit hackish i think.
And we can also avoid exporting the context directly (only provide it through the hook).

Reply:
As discussed not going to change this for now. :bow: